app/assets/javascripts/arclight/context_navigation.js

```diff
@@ -87,6 +88,16 @@ class ContextNavigation {
       that.parentLi.after(afterDocs.map(newDoc => newDoc.render()).join('')).fadeIn(500);
     }
   }
+
+  // eslint-disable-next-line class-methods-use-this
+  truncateItems() {
```

Comment:
Can we truncate these docs as we add them to the dom in `updateView`?

Reply:
Moved the call for truncateItems into the updateView method